test build warnings: baseservices\exceptions\regressions\Dev11\147911\fpcw.vcxproj

There should be no build warnings.

```
    24>d:\gh\coreclr\tests\src\baseservices\exceptions\regressions\dev11\147911\fpcw.cpp(46): warning C4244: 'initializing': conversion from 'double' to 'float', possible loss of data [D:\gh\coreclr\bin\tests\obj\Windows_NT.x64.Debug\Native\src\baseservices\exceptions\regressions\Dev11\147911\fpcw.vcxproj]
    24>d:\gh\coreclr\tests\src\baseservices\exceptions\regressions\dev11\147911\fpcw.cpp(104): warning C4244: '=': conversion from 'double' to 'float', possible loss of data [D:\gh\coreclr\bin\tests\obj\Windows_NT.x64.Debug\Native\src\baseservices\exceptions\regressions\Dev11\147911\fpcw.vcxproj]
    24>d:\gh\coreclr\tests\src\baseservices\exceptions\regressions\dev11\147911\fpcw.cpp(160): warning C4244: '=': conversion from 'double' to 'float', possible loss of data [D:\gh\coreclr\bin\tests\obj\Windows_NT.x64.Debug\Native\src\baseservices\exceptions\regressions\Dev11\147911\fpcw.vcxproj]
    24>d:\gh\coreclr\tests\src\baseservices\exceptions\regressions\dev11\147911\fpcw.cpp(46): warning C4189: 'f': local variable is initialized but not referenced [D:\gh\coreclr\bin\tests\obj\Windows_NT.x64.Debug\Native\src\baseservices\exceptions\regressions\Dev11\147911\fpcw.vcxproj]
    24>d:\gh\coreclr\tests\src\baseservices\exceptions\regressions\dev11\147911\fpcw.cpp(103): warning C4101: 'x': unreferenced local variable [D:\gh\coreclr\bin\tests\obj\Windows_NT.x64.Debug\Native\src\baseservices\exceptions\regressions\Dev11\147911\fpcw.vcxproj]
    24>d:\gh\coreclr\tests\src\baseservices\exceptions\regressions\dev11\147911\fpcw.cpp(159): warning C4101: 'x': unreferenced local variable [D:\gh\coreclr\bin\tests\obj\Windows_NT.x64.Debug\Native\src\baseservices\exceptions\regressions\Dev11\147911\fpcw.vcxproj]
```

--- a/tests/src/baseservices/exceptions/regressions/Dev11/147911/fpcw.cpp
+++ b/tests/src/baseservices/exceptions/regressions/Dev11/147911/fpcw.cpp
@@ -45,9 +45,10 @@ extern "C" FPCW_API int RaiseFPException(void)
     __try
     {
         float d = 2.0 - (0.0+2.0);
-        float f = 2.0/d;
-        printf("Shouldnt execute!\n");
-	return 99;
+        float f = 2.0f/d;
+        printf("Shouldn't execute!\n");
+        (void)f; // nop to disable warning C4189
+        return 99;
     }
     __except((GetExceptionCode()==0xc000008e)?EXCEPTION_EXECUTE_HANDLER:EXCEPTION_CONTINUE_SEARCH)
     {
@@ -101,12 +102,12 @@ extern "C" FPCW_API int RaiseFPException(void)
    {
     __try
     {
-         double a = 1e-40, b;
-	 float  x, y;
-	y = a;
+        double a = 1e-40, b;
+        float  y;
+        y = (float)a;
         b = 2.0;
-        printf("Shouldnt execute!\n");
-	return 98;
+        printf("Shouldn't execute!\n");
+        return 98;
     }
     __except((GetExceptionCode()==0xc0000093)?EXCEPTION_EXECUTE_HANDLER:EXCEPTION_CONTINUE_SEARCH)
     {
@@ -157,12 +158,12 @@ extern "C" FPCW_API int RaiseFPException(void)
    {
     __try
     {
-         double a = 1e+40, b;
-	 float  x, y;
-	y = a;
+        double a = 1e+40, b;
+        float  y;
+        y = (float)a;
         b = 2.0;
-        printf("Shouldnt execute!\n");
-	return 97;
+        printf("Shouldn't execute!\n");
+        return 97;
     }
     __except((GetExceptionCode()==0xc0000091)?EXCEPTION_EXECUTE_HANDLER:EXCEPTION_CONTINUE_SEARCH)
     {

